WordPress/Classes/ViewRelated/Views/List/NotificationsList/AvatarsView.swift

```diff
@@ -0,0 +1,144 @@
+import SwiftUI
+import DesignSystem
+
+struct AvatarsView: View {
```

Comment:
[Suggestion] The previous `avatar` configuration handles Gravatar images. I think the new component should maintain this behavior. What do you think?

![CleanShot 2024-02-08 at 17 48 48](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/d1296ce5-19ce-47d3-aa7d-0e56e01ea9eb)

Reply:
Resolved in: #https://github.com/wordpress-mobile/WordPress-iOS/pull/22564/commits/4c974bec039656249576cbf2efbaf8b524b7915b